styleguide/src/theme.ts

```diff
@@ -10,10 +10,10 @@ declare module '@material-ui/core/styles/createPalette' {
   }
 
   interface PaletteOptions {
-    success: PaletteColorOptions
-    warning: PaletteColorOptions
-    listPendingStatus: ListStatus
-    listCompletedStatus: ListStatus
+    success?: PaletteColorOptions
```

Comment:
@HenryNguyen5 Without making these optional _**here**_, I was not able to bypass the errors in the other `theme.ts`. After adding `const theme: ThemeOptions` typescript complained `success, warning, listPendingStatus, listCompletedStatus` were not defined in explorer's `theme.ts`.

Reply:
@rupurt is there a need for two `theme.ts` files? currently, `explorer` has one and `styleguide` has the other one,  and they are very much alike. can we delete `explorer` `theme.ts`?

Reply:
committed in https://github.com/smartcontractkit/chainlink/pull/2119/commits/c057b575a130452ad40fd8c3b30aad5af9dd4204 for demonstration

Reply:
@NavyAdmiral good call on getting them combined together. `c057b57` looks good besides the copy + pasta ✂️ comment

Reply:
That's an artifact from an old commit, the new one is rebased into https://github.com/smartcontractkit/chainlink/pull/2119/commits/33eb8f8fd7adb031aed69a7191d5d917a475ef2b

Reply:
lets resolve this then, and it was @HenryNguyen5's who pointed out the 2 themes, so thanks @HenryNguyen5 :shipit: